.github/ISSUE_TEMPLATE/election.md

```diff
@@ -0,0 +1,128 @@
+---
+name: Community Role Election
+about: Nominate candidates for community roles such as Committer, Maintainer, etc.
+labels: election, community
+---
+
+## 🗳️ Community Role Nomination
+
+Please use this template to nominate candidates for community roles such as Committer, Maintainer, etc.
+
+### 📋 Election Title
+
+<!-- Example: Maintainer Nomination for @username -->
+
+### 👤 Nominator Information
+
+- **GitHub Username**:
+- **Name**:
+- **Email**:
+- **Company/Organization**:
+
+### 🎯 Nominee Information
+
+- **GitHub Username**: @
+- **Name**:
+- **Email**:
+- **Company/Organization**:
+
+### ⏰ Nomination/Voting Timeline
+
+- **Nomination Period**: <!-- Example: 2025-08-29 to 2025-09-05 (minimum 7 days) -->
+- **Voting Period**: <!-- Example: 2025-09-06 to 2025-09-13 (minimum 7 days) -->
+- **Results Announcement**: <!-- Example: 2025-09-14 -->
+
+### ✅ Nominee Eligibility Requirements
+
+- [ ] GitHub account has two-factor authentication (2FA) enabled
+- [ ] Active participation in the project over the past three months (code, reviews, discussions, meetings, etc.)
+- [ ] Participated in at least one bi-weekly community meeting
+- [ ] Understands and agrees to follow the [Jeandle Code of Conduct](../../CODE_OF_CONDUCT.md)
+- [ ] Has read and understands the [Community Membership Guidelines](../../COMMUNITY_MEMBERSHIP.md)
+- [ ] Has read and understands the [Contributor Ladder](../../CONTRIBUTOR_LADDER.md)
+- [ ] Possesses the technical skills and experience required for the role
+- [ ] Able to commit the necessary time and responsibility for community duties
+
+### 📝 Nominator Checklist
+
+- [ ] Has communicated with the nominee and received their consent
+- [ ] Information provided is complete and accurate
+- [ ] Clear reasons for nomination and contribution history provided
+- [ ] Confirmed that the nominee has time and willingness to take on the responsibilities
+- [ ] Has informed the nominee about the election process and requirements
+
+### ℹ️ Additional Information
+
+- Election process and role descriptions can be found in the relevant repository documentation
+- Please provide additional comments in this issue for any objections or special circumstances
+- During the voting period, all Maintainers and Committers have voting rights
+- Election results require a **simple majority** (over 50%) to pass
```

Comment:
According to our current rule. Only maintainers have voting rights and two-thirds (rounded up) approvals are required